lib/bundler/cli.rb

```diff
@@ -285,6 +285,7 @@ def show(gem_name = nil)
     if Bundler.feature_flag.list_command?
       desc "list", "List all gems in the bundle"
       method_option "name-only", :type => :boolean, :banner => "print only the gem names"
+      method_option "paths", :type => :boolean, :banner => "print the path to each gem in the bundle"
```

Comment:
what happens if you pass `bundle list  --paths --name-only`?

Reply:
Bundler will just print the same output as `bundler list --name-only`

Reply:
we probably should make it an error if both are passed, then